include/config/item.h

```diff
@@ -26,4 +26,7 @@
 #define VAR_LAST_REPEL_LURE_USED    0       // If this var has been assigned, last Repel/Lure used will be saved and the player will get prompted with the vanilla repel YES/NO option, unless I_REPEL_LURE_MENU is set to TRUE.
 #define I_REPEL_LURE_MENU           TRUE    // If TRUE, the player is able to choose which Repel/Lure to use once the previous one runs out. Cursor position is saved by VAR_LAST_REPEL_LURE_USED if not 0.
 
+// Vs. Seeker
+#define I_VS_SEEKER_CHARGING        0     // If this flag is assigned, the Vs Seeker will functionlity will be enabled. When the player has the Vs. Seeker, Match Call rematch functions will stop working.
```

Comment:
words

Reply:
Before I do this, I want to be super clear - are you saying that anything related to the Vs. Seeker should be wrapped in ifdef to prevent compilation if the flag isn't set?

Reply:
Fixed in https://github.com/PokemonSanFran/pokeemerald-expansion/commit/e4fe0e19eff65df5e93af055551d598e7ff4fcb8